src/plugins/plot/stackedPlot/StackedPlotItem.vue

```diff
@@ -167,39 +166,13 @@ export default {
         }
       }
     },
-
-    updateComponentProp(prop, value) {
-      if (this.component) {
-        this.component[prop] = value;
-      }
-    },
     updateView() {
-      if (this._destroy) {
-        this._destroy();
-        this.component = null;
-        this.$el.innerHTML = '';
-      }
-
-      const onYTickWidthChange = this.onYTickWidthChange;
-      const onLockHighlightPointUpdated = this.onLockHighlightPointUpdated;
-      const onHighlightsUpdated = this.onHighlightsUpdated;
-      const onConfigLoaded = this.onConfigLoaded;
-      const onCursorGuideChange = this.onCursorGuideChange;
-      const onGridLinesChange = this.onGridLinesChange;
-
-      const openmct = this.openmct;
-      const path = this.path;
-
       //If this object is not persistable, then package it with it's parent
       const object = this.getPlotObject();
-
-      const getProps = this.getProps;
-      const isMissing = openmct.objects.isMissing(object);
+      this.isMissing = this.openmct.objects.isMissing(object);
 
       if (this.openmct.telemetry.isTelemetryObject(object)) {
-        this.subscribeToStaleness(object, (stalenessResponse) => {
-          this.updateComponentProp('isStale', stalenessResponse.isStale);
-        });
+        this.subscribeToStaleness(object);
```

Comment:
we don't need to set with the callback since the mixin already provides and updates `isStale`